server/server.js

```diff
@@ -14,10 +14,14 @@ dotenv.config({path: path.join(__dirname, './config/config.env' )})
 
 console.log('request came in here!');
 
-app.use((req, res, next) => {
-    res.header('Access-Control-Allow-Origin', '*');
-    next();
-  });
+app.all('*', function(req, res, next) {
```

Comment:
Nonblocking: This is awesome, would you add some comments so the entire team is clear what this does?